documentation/proc-pages/development/standards.md

```diff
@@ -163,6 +164,20 @@ It is greatly encouraged and recommended to include type hints for all inputs an
 
 ## Docstrings
 
+The docstring style is that of the [Google type](https://google.github.io/styleguide/pyguide.html#s3.8.1-comments-in-doc-strings). Though there are some additions for `Notes` and `References` in order to give mathematical reasoning and sources to some functions.
```

Comment:
I really meant not to start including Google-style docstrings when the majority of the codebase already uses sphinx-style. I think ff we want to switch over to another one, this should be done in a separate PR, hopefully using a tool to convert all existing docstrings.

Reply:
I will amend